classes/output/workflowhistory/renderer.php

```diff
@@ -86,8 +86,14 @@ public function render_workflowhistory_table($workflowid, $searchparams = [], $d
 
         $namefields = get_all_user_name_fields(true, 'u');
         $sqlfields = "tfh.*, {$namefields}";
-        $sqlfrom = '{tool_trigger_workflow_hist} tfh LEFT JOIN {user} u ON tfh.userid = u.id';
-        $sqlwhere = 'workflowid = :workflow';
+        $sqlfrom = '{tool_trigger_workflow_hist} tfh';
+
+        $joins = [
+            'LEFT JOIN {user} u ON tfh.userid = u.id',
+        ];
+        $sqlfrom = $sqlfrom . ' ' . implode(' ', $joins);
```

Comment:
This seems overly complicated for the one line. Is there reason to make this extendable currently?

Reply:
Ah, that was leftover from when I was joining additional tables. I can remove this.